Scientist4JCore/src/main/java/com/github/rawls238/scientist4j/Experiment.java

```diff
@@ -25,29 +28,29 @@
     private final Counter totalCount;
 
     public Experiment() {
-      this("Experiment");
+        this("Experiment");
     }
 
     public Experiment(String name) {
         this(name, false);
     }
 
     public Experiment(String name, Map<String, Object> context) {
-      this(name, context, false);
+        this(name, context, false);
     }
 
     public Experiment(String name, boolean raiseOnMismatch) {
-      this(name, new HashMap(), raiseOnMismatch);
+        this(name, new HashMap<>(), raiseOnMismatch);
     }
 
     public Experiment(String name, Map<String, Object> context, boolean raiseOnMismatch) {
-      this.name = name;
-      this.context = context;
-      this.raiseOnMismatch = raiseOnMismatch;
-      controlTimer = metrics.timer(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "control"));
-      candidateTimer = metrics.timer(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "candidate"));
-      mismatchCount = metrics.counter(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "mismatch"));
-      totalCount = metrics.counter(MetricRegistry.name(NAMESPACE_PREFIX, this.name, "total"));
+        this.name = name;
```

Comment:
what's with all the spacing changes? Could you change your IDE to have two spaces for tabs instead of four? IIRC this should have two spaces for tab, not four as I think this tries to introduce